fanuc_r2000ic_support/launch/robot_state_visualize_r2000ic125l.launch

```diff
@@ -17,6 +17,12 @@
 
   <rosparam command="load" file="$(find fanuc_r2000ic_support)/config/joint_names_r2000ic125l.yaml" />
 
+  <include file="$(find fanuc_driver)/launch/robot_state.launch">
+    <arg name="robot_ip"   value="$(arg robot_ip)" />
+    <arg name="J23_factor" value="$(arg J23_factor)" />
+    <arg name="use_bswap"  value="$(arg use_bswap)" />
+  </include>
+
```

Comment:
Are you guys using some other driver?

Reply:
no, but we are not using this launchfile. I still wonder where it was incorrectly copied from though

Reply:
(and yes in a sense as we're using `fanuc_driver_exp` by you but that's robot side only ...)

Reply:
fixed it in 165f as well